crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -122,6 +123,51 @@ pub async fn invoke_map_contract(key: &str, value: &str, account: &str, contract
     execution.send().await.unwrap();
 }
 
+pub async fn invoke_udc_contract(
```

Comment:
I'd rather this could be used as general `invoke_contract` helper function, that takes an address, salt, uniqueness etc as arguments. That way this will be more reusable and won't have hardcoded values.